### ISSUE TITLE
Why's `setMessageHistory` updater function used as dependency in the example code?

### ISSUE THREAD

**Issue by Nisthar** [robtaussig/react-use-websocket#219]
```js
const [messageHistory, setMessageHistory] = useState([]);

  const { sendMessage, lastMessage, readyState } = useWebSocket(socketUrl);

  useEffect(() => {
    if (lastMessage !== null) {
      setMessageHistory((prev) => prev.concat(lastMessage));
    }
  }, [lastMessage, setMessageHistory]);
  ```
  What's the point of passing the `setMessageHistory` function as a dependency?

**Comment by cornejobarraza** [comment]
Linters usually scream at you if you don't pass `setState` as a dependency, that might be the reason he did it. React docs (only found this in legacy) says you can omit it as a dependency, so passing it shouldn't be an issue either:

https://legacy.reactjs.org/docs/hooks-reference.html#usestate

**Comment by robtaussig** [comment]
Fixed in [b12af6d](https://github.com/robtaussig/react-use-websocket/commit/b12af6d2ebdffab37691c99e7314e6701fbccb10)
